web/modules/custom/usa_twig_vars/usa_twig_vars.module

```diff
@@ -351,22 +351,18 @@ function usa_twig_vars_after_build($form, &$form_state) {
 }
 
 /** Helper function to check if node should be excluded from contact center based on path. Updates the exclude from url checkbox on edit pages.  */
-function usa_twig_vars_check_exclude_from_contact_center($nodeToCheck) {
-  $pathAliasToCheck = $nodeToCheck->toUrl()->toString();
-  if (str_contains($pathAliasToCheck, '/es/articulos/') || str_contains($pathAliasToCheck, '/features/')) {
-    $old_checkbox = $nodeToCheck->get('field_exclude_from_contact_cente')[0]->value;
-
-    $nodeToCheck->set('field_exclude_from_contact_cente', 1);
-    $nodeToCheck->save();
-
-    $new_checkbox = $nodeToCheck->get('field_exclude_from_contact_cente')[0]->value;
+function usa_twig_vars_check_exclude_from_contact_center($node_to_check) {
+  $path_alias_to_check = $node_to_check->toUrl()->toString();
+  if (str_contains($path_alias_to_check, '/es/articulos/') || str_contains($path_alias_to_check, '/features/')) {
```

Comment:
If you're strictly interested in paths that start with either of these, you could use [str_starts_with()](https://www.php.net/str_starts_with) and prevent matches with something like `/unrelated-program/about/features/` from matching here.